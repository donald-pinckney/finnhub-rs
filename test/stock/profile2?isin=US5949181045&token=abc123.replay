{"request":{"url":"https://finnhub.io/api/v1/stock/profile2?isin=US5949181045&token=abc123","method":"GET","body":null,"headers":{}},"response":{"url":"https://finnhub.io/api/v1/stock/profile2?isin=US5949181045&token=abc123","status":200,"headers":{"access-control-allow-credentials":"true","access-control-allow-headers":"Origin","access-control-allow-methods":"POST, GET, OPTIONS, PUT, DELETE","access-control-allow-origin":"*","access-control-expose-headers":"Content-Length","alt-svc":"h3-27=\":443\"; ma=86400, h3-28=\":443\"; ma=86400, h3-29=\":443\"; ma=86400","cf-cache-status":"DYNAMIC","cf-ray":"644c3cf7399b040c-ORD","cf-request-id":"09a3846e7e0000040c26811000000001","connection":"keep-alive","content-type":"application/json; charset=utf-8","date":"Sat, 24 Apr 2021 03:27:50 GMT","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","nel":"{\"report_to\":\"cf-nel\",\"max_age\":604800}","report-to":"{\"max_age\":604800,\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report?s=l9D5K2wj99bRrqjL8bQTaSz8ep1q2ChpHE9cJUJ9Z%2F9Fqw46VB9UsyJKrdUBH25Tmv8fsvAAWsmzjyTR7sSFVTREduWSPwDJQdkt9CP1ePk%2Bnp2fluDr\"}],\"group\":\"cf-nel\"}","server":"cloudflare","set-cookie":"__cfduid=db11f98a29e081a70e7881ac872c8340e1619234870; expires=Mon, 24-May-21 03:27:50 GMT; path=/; domain=.finnhub.io; HttpOnly; SameSite=Lax; Secure","transfer-encoding":"chunked","x-ratelimit-limit":"60","x-ratelimit-remaining":"54","x-ratelimit-reset":"1619234930"},"body":"eyJjb3VudHJ5IjoiVVMiLCJjdXJyZW5jeSI6IlVTRCIsImV4Y2hhbmdlIjoiTkFTREFRIE5NUyAtIEdMT0JBTCBNQVJLRVQiLCJmaW5uaHViSW5kdXN0cnkiOiJUZWNobm9sb2d5IiwiaXBvIjoiMTk4Ni0wMy0xMyIsImxvZ28iOiJodHRwczovL2Zpbm5odWIuaW8vYXBpL2xvZ28/c3ltYm9sPU1TRlQiLCJtYXJrZXRDYXBpdGFsaXphdGlvbiI6MTk2NTM1MSwibmFtZSI6Ik1pY3Jvc29mdCBDb3JwIiwicGhvbmUiOiIxNDI1ODgyODA4MC4wIiwic2hhcmVPdXRzdGFuZGluZyI6NzU0Mi4yMTU3NjcsInRpY2tlciI6Ik1TRlQiLCJ3ZWJ1cmwiOiJodHRwczovL3d3dy5taWNyb3NvZnQuY29tL2VuLXVzIn0="},"format_version":3}