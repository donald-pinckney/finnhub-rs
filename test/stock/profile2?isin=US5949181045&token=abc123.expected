CompanyProfile {
    country: "US",
    currency: "USD",
    exchange: "NASDAQ NMS - GLOBAL MARKET",
    ipo: "1986-03-13",
    name: "Microsoft Corp",
    phone: "14258828080.0",
    shareOutstanding: 7542.215767,
    ticker: "MSFT",
    weburl: "https://www.microsoft.com/en-us",
    logo: "https://finnhub.io/api/logo?symbol=MSFT",
    finnhubIndustry: "Technology",
}
